elytron/pom.xml

```diff
@@ -189,6 +189,11 @@
             <artifactId>wildfly-elytron-sasl</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.wildfly.security</groupId>
+            <artifactId>wildfly-elytron-sasl-auth-util</artifactId>
+        </dependency>
```

Comment:
Just to check, is anything making use of this? I don't see any imports adjusted related to this one.

Reply:
Yes, I have checked no imports are dependent on this dependency. As I remember some test case was failing in wildly but I have re-checked everything is fine. I will push the updated one again.